harness/tests/experiment/keras/test_keras_data.py

```diff
@@ -157,13 +157,12 @@ def test_sampler(shuffle: bool, skip: int, rank_size: Tuple[int, int]) -> None:
     assert got_indices == expect_indices[: len(got_indices)]
 
 
-@pytest.mark.parametrize("use_multiprocessing", [False, True])
 @pytest.mark.parametrize("workers", [0, 1, 5])
 @pytest.mark.parametrize("rank_size", [(0, 1), (0, 3), (1, 3), (2, 3)])
 @pytest.mark.parametrize("skip", [0, 50, 350])
 @pytest.mark.parametrize("shuffle", [False, True])
-def test_enqueuer(
-    shuffle: bool, skip: int, rank_size: Tuple[int, int], workers: int, use_multiprocessing: bool
+def test_enqueuer_threading(
```

Comment:
I believe `test_build_enqueuer` is probably more appropriate here given the parameter sweep.